.github/workflows/kubevirt-ipam-controller.yaml

```diff
@@ -0,0 +1,23 @@
+name: Kubevirt IPAM controller Tests
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version-file: 'go.mod'
```

Comment:
btw this is tricky
it takes the one from CNAO's go.mod
and use it as the go version for OVN-K as well

for now it works, it might break in the future and then we will take care about it